crates/cheatnet/src/constants.rs

```diff
@@ -52,6 +52,8 @@ pub const TEST_CONTRACT_CLASS_HASH: &str = "0x117";
 pub const TEST_ADDRESS: &str = "0x01724987234973219347210837402";
 
 pub const STEP_RESOURCE_COST: f64 = 0.01_f64;
+// snforge_std/src/cheatcodes.cairo::TEST
+pub const TEST_MEMORY_ADDRESS: &str = "0x01724987234973219347210837402";
```

Comment:
We have it defined about as `TEST_ADDRESS` xD

Reply:
I moved the definition to `cheatnet`